fieldservice/models/fsm_order.py

```diff
@@ -0,0 +1,143 @@
+# Copyright (C) 2018 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from datetime import timedelta
+from odoo import api, fields, models, _
+from . import fsm_stage
+
+
+class FSMOrder(models.Model):
+    _name = 'fsm.order'
+    _description = 'Field Service Order'
+    _inherit = ['mail.thread', 'utm.mixin', 'rating.mixin',
+                'mail.activity.mixin', 'portal.mixin']
+
+    def _default_stage_id(self):
+        return self.env.ref('fieldservice.fsm_stage_new')
```

Comment:
It's best to get the first available stage in the list. Otherwise deleting this record would break the module.